docs/03.API-EXAMPLE.md

```diff
@@ -283,6 +284,12 @@ main (int argc, char * argv[])
       len++;
     }
 
+    /* If the command starts with 'quit', break the loop */
+    if (!strncmp (cmd, "quit", strlen ("quit")))
```

Comment:
Yes, because the `cmd` contains a `\n` at the end and the `strcmp` doesn't find them equal, so this is the reason why the `strncmp` is used instead of `strcmp`.